wso2-synapse-unit-test-plugin/src/main/java/org/wso2/synapse/unittest/UnitTestClient.java

```diff
@@ -0,0 +1,75 @@
+/*
+ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.synapse.unittest;
+
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.plugin.logging.SystemStreamLog;
+
+/**
+ * SynapseTestCase file read class in unit test framework.
+ */
+class UnitTestClient {
+
+    private static Log log;
+
+    private UnitTestClient() {
+    }
+
+    /**
```

Comment:
Shall we add proper java doc comments with block tags such as @param, @return etc.?

Reply:
Added java docs for missing methods